modules/gcp/win-gfx/win-gfx-startup.ps1.tmpl

```diff
@@ -21,6 +21,10 @@ $DATA.Add("pcoip_registration_code", "${pcoip_registration_code}")
 $DATA.Add("admin_password", "${admin_password}")
 $DATA.Add("ad_service_account_password", "${ad_service_account_password}")
 
+$ENABLE_AUTO_SHUTDOWN     = "${enable_workstation_idle_shutdown}"

```

Comment:
not familiar enough to know if "true" or "True" is what you want - you want a boolean so would it be
$ENABLE_AUTO_SHUTDOWN = $${enable_workstation_idle_shutdown}
so that after the template is rendered, it would become $true or $false?